src/OrbitGl/Track.cpp

```diff
@@ -19,10 +17,17 @@
 #include "TimeGraphLayout.h"
 #include "Viewport.h"
 
+using orbit_client_data::TrackData;
+
 Track::Track(CaptureViewElement* parent, TimeGraph* time_graph, orbit_gl::Viewport* viewport,
              TimeGraphLayout* layout, const orbit_client_model::CaptureData* capture_data,
              uint32_t indentation_level)
     : CaptureViewElement(parent, time_graph, viewport, layout),
+      num_prioritized_trailing_characters_{0},
```

Comment:
better moving to the constuctor instead than in the .h file? Why?

Reply:
well, I was following clang-tidy on this one. :)